Tasks/Common/webdeployment-common-v2/jsonvariablesubstitutionutility.ts

```diff
@@ -97,12 +101,14 @@ export function substituteJsonVariableV2(jsonObject, envObject) {
                         console.log(tl.loc('SubstitutingValueonKeyWithString' , jsonChild , resultNode.value));
                         jsonObject[jsonChild] = resultNode.value;
                 }
+                isValueChanged = true;
```

Comment:
check with @N-Usha if we need to fail when we find key but value is already set. Or just fail if key is not there.